src/docfx/config/Config.cs

```diff
@@ -3,22 +3,26 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Docs.Build
 {
     internal class Config
     {
+        private static readonly string[] s_defaultContentInclude = new[] { "docs/**/*.{md,yml,json}" };
+        private static readonly string[] s_defaultContentExclude = Array.Empty<string>();
+
         /// <summary>
         /// Gets the default locale of this docset.
         /// </summary>
         public readonly string Locale = "en-us";
 
         /// <summary>
-        /// Gets the files that are managed by this docset.
+        /// Gets the contents that are managed by this docset.
         /// </summary>
-        public readonly FileConfig Files = new FileConfig();
+        public readonly FileConfig Content = new FileConfig(s_defaultContentInclude, s_defaultContentExclude);
```

Comment:
why move it here?

Reply:
FileConfig may be used for other config like `resource`, `template`, with different default value.

---
In reply to: [184842625](https://github.com/dotnet/docfx/pull/2702#discussion_r184842625) [](ancestors = 184842625)